frontend/reactnative/lib/Auth.js

```diff
@@ -0,0 +1,60 @@
+import Forge from 'node-forge';
+
+import * as Config from './Config.js';
+
+let userKey;
+
+export function wipeUserId(){
+	Config.eraseSetting("userKey").then(function(result) {
+		// log if erase was sucessful or not
+		console.log("Erase sucess: " + result);
+	});
+}
+	
+export async function loadUserId(){
```

Comment:
can you make this function return the key once it's finished?

Reply:
It could return a promise, however, as the function is called in the constructor and it is asymetric there is no way,i know of to wait for the funcion to be complete. Due to this i dont see any benefit of making the function return the promise as it will then have to have a callback specified.